### ISSUE TITLE
Monotonic time.

### ISSUE THREAD

**Issue by digitalextremist** [celluloid/culture#25]
Across the board, remove non-monotonic time. All gems, every case.

/cc: #24

**Comment by tarcieri** [comment]
I think this is already done but I may have missed a few places.

Grepping for Time.now should hopefully reveal any remaining cases

On Thursday, August 6, 2015, Donovan Keme <notifications@github.com> wrote:

> Across the board, remove non-monotonic time. All gems, every case.
>
> /cc: #24 <https://github.com/celluloid/culture/issues/24>
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/celluloid/culture/issues/25>.
>


-- 
Tony Arcieri

**Comment by digitalextremist** [comment]
Yeah, `67` instances ... I'll catch those in one commit after #2
